platforms/jvm/language-jvm/src/test/groovy/org/gradle/api/internal/tasks/compile/MinimalCompilerDaemonForkOptionsConverterTest.groovy

```diff
@@ -32,8 +32,8 @@ class MinimalCompilerDaemonForkOptionsConverterTest extends Specification {
         def javaForkOptions = converter.transform(baseForkOptions)
 
         then:
-        javaForkOptions.minHeapSize == "128m"
-        javaForkOptions.maxHeapSize == "1g"
+        javaForkOptions.minHeapSize.getOrNull() == "128m"
+        javaForkOptions.maxHeapSize.getOrNull() == "1g"
```

Comment:
💅 Wouldn't it make sense to just call `get()` in these tests? We do expect a value, and if there is no value, `get()` would also fail. So ultimately it is a shorter / terser way of saying the same thing.

Reply:
Fixed